clients/js-legacy/src/extensions/extensionType.ts

```diff
@@ -208,6 +223,7 @@ export function getAccountTypeOfMintType(e: ExtensionType): ExtensionType {
         case ExtensionType.GroupMemberPointer:
         case ExtensionType.TokenGroup:
         case ExtensionType.TokenGroupMember:
+        case ExtensionType.PausableConfig:
```

Comment:
And to go with the other one
```suggestion
        case ExtensionType.PausableAccount:
```